templates/zookeeper.yaml

```
@@ -0,0 +1,28 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: zookeeper
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: zookeeper
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.1.0"
+    appversion.kubeaddons.mesosphere.io/zookeeper: "0.2.0"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
```

Technically speaking, since KUDO doesn't support <1.16 kubernetes, this kudo based addon should not claim 1.15.0 support.

```suggestion
    minSupportedVersion: v1.16.0
```